cascalog-more-taps/test/cascalog/more_taps_test.clj

```diff
@@ -40,6 +40,24 @@
             ((hfs-delimited tmp :delimiter "," :compress? true) ?a ?b ?c)) =>
         (produces [["Proin" "false" "3"]])))))
 
+(deftest delimited-compression-test
```

Comment:
I guess this just shows that the tap round trips, not that any compression is going on, yeah?